src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -330,37 +330,28 @@ def refreshCache(self):
                 if nonAbortSites: # if there is at least a non aborted/down site then run there, otherwise fail the job
                     possibleLocations = nonAbortSites
                 else:
-                    newJob['name'] = loadedJob['name']
-                    newJob['possibleLocations'] = possibleLocations
+                    newJob['possibleSites'] = possibleLocations
                     badJobs[71102].append(newJob)
                     continue
 
             # try to remove draining sites if possible, this is needed to stop
             # jobs that could run anywhere blocking draining sites
             # if the job type is Merge, LogCollect or Cleanup this is skipped
-            if newJob['type'] not in self.ioboundTypes:
```

Comment:
where does type comes from I don't see original "type" key switch to "task_type" in the patch.

Reply:
Never mind, I found it from below DAO.

wmbs_sub_types.name AS type,